# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/crashtests/crashtests.list
# Commit: 124efdbe177c
# Full Hash: 124efdbe177c23cb51af54d16e3f1404f5c664ad
# Author: Timothy Nikkel <tnikkel@gmail.com>
# Date: 2025-08-12 09:46:05
# Description:
#   Bug 1981495. In Theme::PaintCircleShadow check if we can create a similar draw target before trying to create one. r=gfx-reviewers,nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D260615
# ==============================================================================

diff -r 43fcb1ed39b3 -r 124efdbe177c widget/crashtests/crashtests.list
--- a/widget/crashtests/crashtests.list	Tue Aug 12 02:12:07 2025 +0000
+++ b/widget/crashtests/crashtests.list	Tue Aug 12 03:43:25 2025 +0000
@@ -2,3 +2,4 @@
 load 303901-2.html
 load 380359-1.xhtml
 load 1128214.html
+load 1981495-1.html
